routing/heap.go

```diff
@@ -21,6 +21,9 @@ type nodeWithDist struct {
 	// amount that includes also the fees for subsequent hops.
 	amountToReceive lnwire.MilliSatoshi
 
+	// Total cltv amount from this node to the destination
```

Comment:
nit: missing period.

Also, should this be `destination` or `source`? I see `source` is used elsewhere in this struct, but I guess we changed to destination at one point.

Reply:
You're right, here source node is the actual destination since we do the path finding in reverse